site/src/docs/getting-started/contributing/how-to-contribute.mdx

```diff
@@ -10,16 +10,30 @@ import LeadParagraph from '../../../components/LeadParagraph';
 import Image from '../../../components/Image';
 import InternalLink from '../../../components/InternalLink';
 
-# Before contributing
+# How to contribute
 
 <LeadParagraph>
-    You can contribute content to the HDS in different ways. Before you start contributing to HDS, please refer to following guidelines.
+    You can contribute to the HDS in many ways. Before you start contributing to HDS, read the following guide so we can make the most out of your contribution.
 </LeadParagraph>
 
-Every component in HDS includes three parts:
-1. Design 
-2. Implementation 
-3. Documentation 
+Ways to contribute to HDS:
+* Discuss HDS in Slack channel #designsystem (Only for City of Helsinki staff)
+* Attend HDS demo events to get to know new features and ask questions, <InternalLink href="/about/#contact">contact</InternalLink> for invite
+* <InternalLink href="/about/#contact">Contact</InternalLink> HDS team to see how we can help
```

Comment:
Does the "we" refer to HDS team or the contributors? Maybe contact HDS team to see how we can help each other?